az-core/src/main/java/azkaban/Constants.java

```diff
@@ -32,6 +32,9 @@
  */
 public class Constants {
 
+  // Azkaban Flow Versions
+  public static final String AZKABAN_FLOW_VERSION_20 = "2.0";
```

Comment:
version_2? or version_2_0?

Reply:
Version_2_0 seems better, I'll change it.